Zend/zend.h

```diff
@@ -77,7 +77,7 @@
 #define ZEND_TSRMLS_CACHE
 #endif
 
-ZEND_TSRMLS_CACHE_EXTERN();
+ZEND_TSRMLS_CACHE_EXTERN()
```

Comment:
This macro requires a semicolon under ZTS. If we want to change this, the semicolon needs to be added to the ZTS macro definition.